crud/readview.lua

```diff
@@ -0,0 +1,613 @@
+local errors = require('errors')
+
+local fiber = require('fiber')
+local common = require('crud.select.compat.common')
+
+local Merger = require('crud.select.merger')
+
+local stash = require('crud.common.stash')
+local utils = require('crud.common.utils')
+local sharding = require('crud.common.sharding')
+local select_executor = require('crud.select.executor')
+local select_filters = require('crud.compare.filters')
+local dev_checks = require('crud.common.dev_checks')
+local schema = require('crud.common.schema')
+local const = require('crud.common.const')
+local sharding_metadata_module = require('crud.common.sharding.sharding_metadata')
+local stats = require('crud.stats')
+local checks = require('checks')
+
+local compare_conditions = require('crud.compare.conditions')
+local select_plan = require('crud.compare.plan')
+
+local ReadviewError = errors.new_class('ReadviewError', {capture_stack = false})
+
+local readview = {}
+
+function checkers.vshard_call_mode(p)
```

Comment:
These three lines add a new function for global `checkers` table, you shouldn't do it one more time.

Reply:
fixed